### ISSUE TITLE
possible error in TD examples (mqtt)

### ISSUE THREAD

**Issue by adriankast** [thingweb/thingweb-playground#158]
Just found this:
```json
{
  "href": "mqtt://iot.eclipse.org/oneOfTest",
  "op":"observeproperty",
  "mqtt:commandCode":8
},
{
  "href": "mqtt://iot.eclipse.org/oneOfTest",
  "op": "unobserveproperty",
  "mqtt:methodCode": 10
}
```
in the `publisher.json` TD example. I think it should be:
```json
{
  "href": "mqtt://iot.eclipse.org/oneOfTest",
  "op":"observeproperty",
  "mqv:controlPacketValue": "SUBSCRIBE"
},
{
  "href": "mqtt://iot.eclipse.org/oneOfTest",
  "op": "unobserveproperty",
  "mqv:controlPacketValue": "UNSUBSCRIBE"
}
```
or deleted since its the defaults. What do you think @egekorkan ?

**Comment by egekorkan** [comment]
Yes the correction is correct :) Old examples...

**Comment by adriankast** [comment]
Okay :) fixed in 2252fd11f030ceb726dd6439d8046682c531fc25

**Comment by adriankast** [comment]
was fixed with #161
